From 56b7844d74efe2b224f85ca2eadcedd729d17360 Mon Sep 17 00:00:00 2001
From: Shawn Xiao <b49994@freescale.com>
Date: Mon, 2 Feb 2015 15:39:51 +0800
Subject: [PATCH 0679/5242] MGS-1276 ENGR00306443-2 dts:Add QoS description in
 imx6sx dts

commit  156a836ebafc120105bd9e623e4c49d43556c2f4 from
https://source.codeaurora.org/external/imx/linux-imx.git

(cherry-pick from f9759787e60ad3422d2119f9f25ac320ca58f5df)

confilict: arch/arm/boot/dts/imx6sx.dtsi
The dts file arch has changed in 3.14. Add QoS description in
imx6sx dts manually and solve the conflict.

Date Feb 2, 2015

Signed-off-by: Shawn Xiao <b49994@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 383d301..f4a5885 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1272,6 +1272,11 @@
 				clocks = <&clks IMX6SX_CLK_I2C4>;
 				status = "disabled";
 			};
+
+			qosc: qosc@21fc000 {
+				compatible = "fsl,imx6sx-qosc";
+				reg = <0x021fc000 0x4000>;
+			};
 		};
 
 		aips3: aips-bus@2200000 {
-- 
1.7.9.5

